services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouterController.java

```diff
@@ -224,15 +224,15 @@ public RouteTableWebJson updateVpcRouteTable(@PathVariable String projectid, @Pa
             List<InternalSubnetRoutingTable> internalSubnetRoutingTableList = new ArrayList<>();
             for (RouteTable routeTable : vpcRouteTables) {
                 String subnetId = routeTable.getOwner();
-                UpdateRoutingRuleResponse updateRoutingRuleResponse = this.neutronRouterService.updateRoutingRule(subnetId, newRouteEntry, true);
+                UpdateRoutingRuleResponse updateRoutingRuleResponse = this.neutronRouterService.updateRoutingRule(subnetId, newRouteEntry, true, true);
```

Comment:
@kevin-zhonghao This is `updateVpcRouteTable`, how can you know it's *add* operation?

Reply:
The flag - "isAddOperation" represents if API/methods that call updateSubnetRouteTable have the requirement of adding or creating. For example, removeInterface / removeRoutes API only have requirement of deletion.
So update operation should set up "isAddOperation" as true.

Reply:
@kevin-zhonghao It's better to give some comments in the API to describe how to use these parameters.